test/chi/custom-elements/tabs.pug

```diff
@@ -12,8 +12,90 @@ h2 Base
 //- .-w--100
 //-   chi-tabs(active-tab='tab-a' border id='base' sliding-border)
 
-//- h2 Vertical
-//- chi-tabs(active-tab='tab-a' id='vertical' vertical sliding-border)
+h2 Horizontal
+h3 Flat 
+h4 Base
+.-w--100 
+  chi-tabs(id='horizontal-flat-base')
+
+h4 Inverse
+.-w--100.-bg--black
+  chi-tabs(id='horizontal-flat-inverse' inverse)
+
+h3 Solid 
+.-w--100 
+  chi-tabs(id='horizontal-flat-solid' solid size="lg" border)
+
+h2 Vertical 
+h3 Flat
+
+h4 Base
+.-w--100
+  chi-tabs(id='vertical-flat-base' vertical)
+
+h4 Inverse
+
+.-w--100.-bg--black
+  chi-tabs(id='vertical-flat-inverse' vertical inverse)
+
+h3 Solid 
+.-w--100 
+  chi-tabs(id='vertical-solid' vertical solid) 
+
+//- not suported by wc
+//- h3 Subtabs 
+//- .-w--100 
+//-   chi-tabs(id='vertical-subtabs' vertical)
+
+h2 Border
+h3 Base
+.-w--100
+  chi-tabs(id='bordered-base' border)
+
+h3 Inverse
+
+.-w--100.-bg--black
+  chi-tabs(id='bordered-inverse' inverse border)
+
+h2 Additional sizes 
+h3 Horizontal
+h4 XS
+.-w--100 
+  chi-tabs(id='size-XS-horizontal' size='xs')
+h4 SM
+.-w--100 
+  chi-tabs(id='size-SM-horizontal' size='sm')
+h4 MD
+.-w--100 
+  chi-tabs(id='size-MD-horizontal' size='md')
+h4 LG
+.-w--100 
+  chi-tabs(id='size-LG-horizontal' size='lg')
+h4 XL
+.-w--100 
```

Comment:
I think `xl` and `sm--2` are not supported in tabs, am I right @jllr ?

Reply:
Good catch!  Changed "sm--2" for "sm". 

`xl` is supported for tabs in `constants/size.ts`